.github/workflows/version-bump.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - name: Set branch value
         id: variables
         run: |
-          echo "::set-output name=BRANCH_NAME::prep-release/${{ github.event.inputs.version_number }}_$GITHUB_RUN_ID"
+          echo "BRANCH_NAME=prep-release/${{ github.event.inputs.version_number }}_$GITHUB_RUN_ID" >> $GITHUB_OUTPUT
```

Comment:
I know it wasn't doing this before, but let's add some curly braces onto this env var for safety.

```suggestion
          echo "BRANCH_NAME=prep-release/${{ github.event.inputs.version_number }}_${GITHUB_RUN_ID}" >> $GITHUB_OUTPUT
```

Reply:
Fixed. Thanks!